src/Config/packages.aget

```diff
@@ -9,13 +9,13 @@
       "DynamoVisualProgramming.WpfUILibrary": "3.0.0-beta6354",
       "DynamoVisualProgramming.ZeroTouchLibrary": "3.0.0-beta6354",
       "DynamoVisualProgramming.DynamoSamples": "3.0.0-beta6354",
-      "Greg": "2.5.0.5464",
-      "GregRevitAuth": "1.3.8591.28369",
+      "Greg": "3.0.0.2955",
+      "GregRevitAuth": "2.0.8696.26508",
       "pythonnet_py38_win": "2.5.1",
       "Newtonsoft.Json": "13.0.1",
       "NUnit": "3.13.3",
       "Prism": "4.1.0",
```

Comment:
@pinzart90 can we remove this?

Reply:
hmm, let me see what happens if I just remove this.

Reply:
ok, I will try to deal with it here:
https://github.com/DynamoDS/DynamoRevit/pull/2990